### PR TITLE
Fix bal format command to be consistant with other cli commands

### LINKED ISSUE #37868: [Bug]: bal format command only works on non-default modules
### Description

`bal format <module_name>` command seems to be working only for the non-default modules. when the default module is provided, the command ends up with the below error message.

```error: couldn't find an existing module by the name: <module_name>```

### Steps to Reproduce

1. Create a new ballerina package with name `foo`
2. Execute `bal format foo` on the package root directory

### Affected Version(s)

_No response_

### OS, DB, other environment details and versions

_No response_

### Related area

-> Formatter

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
Fixes #37868, #38368

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatCmd.java [hunk 1]
```diff
@@ -39,11 +39,17 @@ public class FormatCmd implements BLauncherCmd {
     @CommandLine.Option(names = {"-d", "--dry-run"})
     private boolean dryRun;
 
+    @CommandLine.Option(names = "--module", hidden = true)
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatCmd.java:42]:
```suggestion
    @CommandLine.Option(names = "--module"
```

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatCmd.java [hunk 2]
```diff
@@ -39,11 +39,17 @@ public class FormatCmd implements BLauncherCmd {
...
+    private String moduleName;
+
+    @CommandLine.Option(names = "--file", hidden = true)
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatCmd.java:45]:
```suggestion
    @CommandLine.Option(names = "--file")
```

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java [hunk 3]
```diff
@@ -41,14 +41,14 @@
...
 import java.util.List;
-import java.util.regex.Pattern;
 
 /**
  * Util class for compilation and format execution for formatting CLI tool.
  */
 class FormatUtil {
     static final String CMD_NAME = "format";
     private static final PrintStream outStream = System.err;
+//    private static Path projectPath;
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java:51]: Do we need to keep this commented line?

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java [hunk 4]
```diff
@@ -46,6 +46,13 @@ public class Messages {
 
     private static final String NO_BALLERINA_FILE_OR_MODULE = "couldn't find an existing ballerina file or " +
             "module by the name: ";
+
+    private static final String CANT_ALLOW_BOTH_MODULE_AND_FILE_OPTIONS =
+            "too many options: either use module option or file option";
+
+    private static final String CANT_ALLOW_MODULE_OR_FILE_OPTIONS =
+            "too many options: single file projects cannot accept module or file option";
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:54]:
```suggestion
    private static final String CANT_ALLOW_MODULE_OR_FILE_OPTIONS =
            "'--module' and '--file' options are not allowed on single file projects";
```

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java [hunk 5]
```diff
@@ -46,6 +46,13 @@ public class Messages {
 
     private static final String NO_BALLERINA_FILE_OR_MODULE = "couldn't find an existing ballerina file or " +
             "module by the name: ";
+
+    private static final String CANT_ALLOW_BOTH_MODULE_AND_FILE_OPTIONS =
+            "too many options: either use module option or file option";
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:51]:
```suggestion
            "using '--module' and '--file' options at the same time for selective formatting is not allowed";
```

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java (hunk 4 above)
Comment (NipunaRanasinghe) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:54]: @praneesha WDYT?

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java (hunk 5 above)
Comment (NipunaRanasinghe) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:51]: @praneesha WDYT?

Comment (praneesha) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:51]: How about `Use one of the '--module' or '--file' options for selective formatting."

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java (hunk 4 above)
Comment (praneesha) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:54]: How about `Do not use '--module' and '--file' options in single file projects.`?

Comment (NipunaRanasinghe) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:54]: @praneesha these are error messages and, IINM we haven't used this format for the CLI error messages in other context? ( Correct me if I'm wrong :) )

Comment (praneesha) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:54]: yes. +1 for `"The '--module' and '--file' options are not allowed in single file projects";`

File: misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java (hunk 5 above)
Comment (NipunaRanasinghe) [reply misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/Messages.java:51]: yes, +1

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**